src/postgres/contrib/yb_auh/yb_auh.c

```diff
@@ -255,6 +278,7 @@ yb_auh_main(Datum main_arg) {
 
     pg_collect_samples(auh_sample_time, auh_sample_size);
     tserver_collect_samples(auh_sample_time, auh_sample_size);
+    collect_table_info();
```

Comment:
This shouldn't be called in the background worker, but rather when we query the view for table id metadata. yb_servers is a good example.